### ISSUE TITLE
[BUG] Super Key does not close the menu panel on second key press

### ISSUE THREAD

**Issue by Harry-W-Haines-III** [linuxmint/cinnamon#11057]
```
 * Cinnamon version 5.4.10
 * Distribution - LMDE 5 Elsie
 * Graphics GeForce GT 1030, NVIDIA 470.129.06
 * 64 bit
  ```

### Issue
After some recent Cinnamon updates it was noticed pressing the Super Key brings up the start menu panel but pressing the Super Key a second time does not close it.  Have to use 'esc' now to close the panel.

### Steps to reproduce
Fully update LMDE 5 and press the Super Key.

### Expected behaviour
Pressing the Super Key a second time should close the start menu panel.

### Configuration info, if applicable
https://pastebin.com/P3hQY6yt

**Comment by JosephMcc** [comment]
A simple search can be your friend: https://github.com/linuxmint/cinnamon/issues/11055

**Comment by Harry-W-Haines-III** [comment]
I did do a search before posting but for some reason it didn't come up.  No need to be a smart aleck about it.  I won't waste my time posting bugs for other people anymore.

**Comment by arjungrad** [comment]
Ah yesterday's update broke my super key too. Have you found any solution yet captain?

**Comment by maciek4231** [comment]
@arjungrad 
https://github.com/linuxmint/cinnamon/commit/7724e4146baf8431bc1fb55dce60984e77adef5a
There is a fix already, if you don't want to wait for an update you can modify this lines in /usr/share/cinnamon/js/ui/keybindings.js

**Comment by arjungrad** [comment]
@maciek4231 Thanks for the reply. It worked: As per your recommendation I went to [this](https://github.com/linuxmint/cinnamon/blob/master/js/ui/keybindings.js) link and copied the keybindings.js to my /usr/share/cinnamon/js/ui/keybindings.js did a reboot and things are working as expected.

**Comment by yllekz** [comment]
Just to add some color to the above workaround, you don't need to reboot after updating keybindings.js - simply ALT + F2 and type "R" to restart Cinnamon and the changes will take effect.

**Comment by mtwebster** [comment]
If you're still experiencing this you should update Cinnamon - fixed in 5.4.11
